interpreter/binary/decode.ml

```diff
@@ -448,6 +450,15 @@ let rec instr s =
   | 0xd1 -> ref_is_null
   | 0xd2 -> ref_func (at var s)
 
+  | 0xfc as b1 ->
+    (match op s with
+    | 0x0f -> table_fill (at var s)
+    | 0x10 -> table_size (at var s)
+    | 0x11 -> table_grow (at var s)
```

Comment:
These encodings contradict the table in #29 (first and last comments), which we have used as the basis for the decoder in Firefox.  If possible I'd like these to change here since we're shipping already.  Proposed encodings there are table_grow == 0x0f, table_size == 0x10, table_fill == 0x11.  Ditto the encoder and any other spots.

Reply:
Ah, sorry, I completely forgot about #29. Changed. (Though it's a bit odd that FF would be shipping features for which we don't even have a spec draft?)